Clinical-Genomics/scout
# Fix SNP ids and links in variant page, general report and clinvar submission form

The current situation is that SNP IDs might be multiple. For instance [this variant](https://scout.scilifelab.se/cust003/19143/72f6fe76c1807ccbd6c31884e4574002) has the following SNP ID: `rs150429680;431849`

- The fist part of that string is the dbSNP id: https://www.ncbi.nlm.nih.gov/snp/rs150429680
- The second part part is the ClinVar variation ID: https://www.ncbi.nlm.nih.gov/clinvar/variation/431849

This PR:
- Fixes the ClinVar submission form by replacing the wrong SNP format with the dbSNP part only (fix #2473 )
- Fixes the variant page by splitting the wrong Id into 2 distinct ID with their links to the respective pages
- Fixes in the same way also the general report

**How to test**:
1. Locally, take a SNP variant and in the database replace its dbsnp_id with 'rs150429680;431849'
2. Check that the distinct links on the variant page work
3. Check that the distinct links on the general report work
4. Pin this variant and another, and on master branch and recreate the situation observed in the clinvar bug (https://github.com/Clinical-Genomics/scout/issues/2473#issuecomment-806023821)
5. Make sure that the submission using this branch works as it should

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
